DTS-STN/canadian-dental-care-plan
# AB#3353 -- Created review adult/child information pages for the adult-child flow

### Description
Created review-adult-information and review-child-information for the adult-child apply flow.

review-adult-information:
- same as adult flow
- buttons changed to continue/back

review-child-information:
- child information, child dental insurance, and child dental benefits data
- dynamic button's continue/submit if more than one child

### Related Azure Boards Work Items
[AB#3353](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3353)

### Screenshots (if applicable)
![c1](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/f2fa9370-a142-43ca-a307-6aa941388d0c)
![c2](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/70b57560-0518-4bab-9f57-af7a78bb94cd)


### Checklist

### Test Instructions
Navigate to the review page for adult-child to confirm:
{id}/adult-child/review-adult-information
{id}/adult-child/review-child-information

### Additional Notes
Using dummy data for testing until workflow is completed
